warp10/src/main/java/io/warp10/continuum/gts/GTSHelper.java

```diff
@@ -3551,8 +3477,7 @@ public static boolean isBucketized(GeoTimeSerie gts) {
     
     GeoTimeSerie serie = new GeoTimeSerie(gts.lastbucket, gts.bucketcount, gts.bucketspan, 4);
     serie.setName(gts.getName());
-    Map<String,String> labels = new HashMap<String,String>();
-    labels.putAll(gts.getLabels());
+    Map<String,String> labels = new HashMap<String,String>(gts.getLabels());
```

Comment:
Please rollback to previous code, it is clearer to understand as the putAll is aligned with the following put.